readthedocs/projects/querysets.py

```diff
@@ -136,7 +135,6 @@ def api(self, user=None, detail=True):
 
 class ProjectQuerySet(SettingsOverrideObject):
     _default_class = ProjectQuerySetBase
-    _override_setting = 'PROJECT_MANAGER'
```

Comment:
Removing these is 👍🏼 ! We should just double check that we are not using them in .com since I think we still have some overrides using this old method.

Reply:
Didn't find any in .com nor the ops repos.